ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -221,6 +221,30 @@ export const displayTierBadgeByRequiredModule = (
   );
 };
 
+export const getEraNote = (profile: Profile): string => {
+  const requiredForRT = isEraRequiredForTier(
+    profile,
+    AccessTierShortNames.Registered
+  );
+  const requiredForCT = isEraRequiredForTier(
+    profile,
+    AccessTierShortNames.Controlled
+  );
+  let note = `*eRA Commons requirements vary by institution. This user's institution 
+  (${profile.verifiedInstitutionalAffiliation.institutionDisplayName}) `;
+
+  if (!requiredForRT && !requiredForCT) {
+    note += 'does not require eRA Commons.';
+  } else if (requiredForRT && !requiredForCT) {
+    note += 'requires eRA Commons for RT access.';
```

Comment:
nit: I think I'd prefer the full "Registered Tier" instead.  What do you think?

Reply:
Yes, just like the module name in table so it's clearer. 
<img width="1129" alt="Screen Shot 2022-02-01 at 10 35 25 AM" src="https://user-images.githubusercontent.com/35533885/151999366-e9c4840a-f2c5-46fb-a450-892c08f3fb0b.png">